OlinkAnalyze/vignettes/bridging_E3072toEHT.Rmd

```diff
@@ -0,0 +1,682 @@
+```
+
+Prior to assessment, outlier bridging samples are excluded. A sample is
+considered an outlier if the NPX value is more than 3 times the
+interquartile range above or below the median on either product.
+
+After assessment, an assay is considered bridgeable if it meets the
+first three criteria. The fourth criteria determines which normalization
+method is recommended for bridging. If all four criteria are met then
+the recommended method is normalization using the median of paired
+differences. If only the first three criteria are met then quantile
+smoothing is recommended. If any of the first three criteria are not met
+then bridging is not recommended for that assay. Note that bridgeable
+assays will differ between projects based on the expression of bridge
+samples in the studies.
+
+### Normalization using the median of paired differences
+
+If it is expected that both the kind of distribution and the variance
+per test between runs are the same, then normalization using the median
+of paired differences will be preferred. Normalization using the median
+of paired differences based on the bridging samples is performed in the
+following steps:
+
+1.  For each assay in the Explore 3072 project, the pairwise difference
+    is calculated for each of the bridging samples with the Explore HT
+    project.
+
+2.  The normalization factor is estimated for each assay by finding the
+    median of the pairwise differences.
+
+3.  The assay-specific normalization factor for each assay is used to
+    normalize each data point from Explore 3072 to Explore HT.
+
+### Quantile smoothing
+
+Since Explore HT and Explore 3072 are two distinct products with
+different workflows involved in generating NPX data, some of the assays
+exist in corresponding but distinct NPX spaces. For those assays, the
+median of paired differences is insufficient for bridging as it only
+considers one anchor point (the median/50% quantile). Instead, quantile
+smoothing (QS) using multiple anchor points (5%, 10%, 25%, 50%, 75%, 90%
+and 95% quantiles) is favored to map the Explore 3072 data to the
+Explore HT distribution. The normalization using QS uses bridging samples
+to perform the following steps:
+
+1.  Each data point of the samples from Explore 3072 is mapped to the
+    equivalent space in Explore HT using an empirical cumulative
+    distribution function. An empirical cumulative distribution function
+    is a probability model which uses the observed data, in this case
+    the NPX values of the bridging samples for an assay, to create a step
+    function which interpolates linearly between the available data
+    points.
+
+2.  The empirical distribution function is used to map the data points
+    from Explore 3072 to the Explore HT space using the specified
+    quantiles. At this point all data points from the bridging samples
+    have NPX values that are normalized to the data points in Explore
+    HT.
+
```

Comment:
Add comma on line 466 after "to normalize the remaining data"

Reply:
Confirming that Explore 3072 should be referenced twice in the bolded section: 

> To normalize the remaining data a spline regression model is constructed **using the sorted Explore 3072 data and the mapped Explore 3072 data**

If this refers to the Explore 3072 data only, would it be clearer to say "using the sorted and mapped Explore 3072 data?

Reply:
Ive reworded this to be clearer: To normalize the remaining data, a spline regression model is
    constructed using the sorted Explore 3072 data (prior to mapping) and the mapped
    Explore 3072 data, along with the anchor points of the spline
    function.